specs/protocol/tx_format.md

```diff
@@ -408,6 +413,9 @@ This output type indicates that the output's amount and owner may vary based on
 
 ## TXPointer
 
+The location of the transaction in the block. It can be used by utxos
+as a reference to the transaction or by the transaction itself to make it unique.
```

Comment:
```suggestion
The location of the transaction in the block. It can be used by UTXOs as a reference to the transaction or by the transaction itself to make it unique.
```